_posts/2020-02-28-status-update.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "28th Feb 2020: IAC Team Status Update"
+author: michaeltlombardi
+categories:
+  - team
+  - status
+tags:
+  - module updates
+  - community
+  - litmus
+---
+
+The IAC team has been working on a lot of interesting things over the past week.
+Here's a brief overview of what we've been up to:
+
+## Updates to Puppet Modules
+
+No modules were released since last week.
+
+## Community Contributions
+
+Thanks to [@jmangt](https://github.com/jmangt) for their improvements to the provision module,
+[adding support for sssh'ing to vagrant boxes where root is not allowed to ssh](https://github.com/puppetlabs/provision/pull/103) and [ensuring the vagrant provision task works against paths with spaces in them](https://github.com/puppetlabs/provision/pull/101).
+
+<!-- ## Approved Modules
+
+This week we happily approved two new modules:
+
+- [fervid/secure_linux_cis](https://forge.puppet.com/fervid/secure_linux_cis) is a CIS hardening module for Linux servers by [fervid.us](https://www.fervid.us/).
+
+- [stackstorm/st2](https://forge.puppet.com/stackstorm/st2) installs, configures and manages the eponymous [StackStorm](https://stackstorm.com) platform for event-driven runbook automation.
+
+[Puppet Approved](https://forge.puppet.com/approved) modules are recommended by us for use with Puppet Open Source and Puppet Enterprise and meet our expectations for quality and usability. -->
+
+## Weekly Test Runs
+
+Numerous PRs went through this week to ensure 44 modules (all those supported modules which have been converted to Litmus so far) run weekly tests via GitHub Actions.
+These tests run against the master branch executing the full test suite - all of the spec tests and then all of the acceptance tests against the full matrix of operating systems for that module (defined in the `release_checks` section of the `provision.yaml` for that module).
+
+As with the rest of our automated tests, these results are sent to Honeycomb for introspection later;
+these test results are also recorded to the [Community Dashboard](https://puppetlabs.github.io/community_management/) in the `Github Actions Report` tab, which shows the test results for the last five runs for each workflow and module.
+Right now, these results are recorded fairly simply, listing the run number, the branch the tests ran against, and whether they passed or failed.
+In the future, we will be improving the usefulness of this dashboard in numerous ways, but for now we just wanted a way to get some quick visualization for triage.
+
+## Facter 4 Incoming
+
+We started doing some prep work this week as [Facter](https://github.com/puppetlabs/facter-ng)'s next major update has [become available as prerelease](https://rubygems.org/gems/facter/versions/4.0.8.pre).
+Facter 4 is a return to Ruby from C++ which should mean easier integration with the rest of our modules and tooling but also means we've got some checking and verifying to do as this major version rolls out.
```

Comment:
I wasn't sure what to highlight/cover here as I didn't work on any of this and didn't see much on it in scrollback.